.github/workflows/installers-conda.yml

```diff
@@ -272,14 +280,20 @@ jobs:
         run: |
           $PKG_PATH -b
 
-          shortcut_path=$HOME/.local/share/applications/spyder_spyder.desktop
+          root_prefix=$(compgen -G $HOME/.local/spyder-*)
+
+          echo "Install info:"
+          echo "Contents of ${root_prefix}:"
+          ls -al $root_prefix
+          echo -e "\nContents of ${root_prefix}/uninstall-spyder.sh:"
+          cat ${root_prefix}/uninstall-spyder.sh
+
+          shortcut_path=/.local/share/applications/spyder_spyder.desktop
+          [[ -e ${root_prefix}/.nonadmin ]] && shortcut_path=${HOME}${shortcut_path} || shortcut_path=/usr${shortcut_path}
```

Comment:
I'm not sure this logic is correct for Linux. If I understand things correctly, with this change we'd install our desktop file to `/usr/.local/share/applications/spyder_spyder.desktop` for admin installations.

But the location any application I know use is `/usr/share/applications`. So, I think this code should be changed to use that path instead.

Reply:
That's correct. I made a transcription error.